src/NetAnalyzers/VisualBasic/Microsoft.NetCore.Analyzers/Runtime/BasicPreferStreamAsyncMemoryOverloads.Fixer.vb

```diff
@@ -59,5 +59,38 @@ Namespace Microsoft.NetCore.VisualBasic.Analyzers.Runtime
 
         End Function
 
+        Protected Overrides Function IsPassingZeroAndBufferLength(bufferValueNode As SyntaxNode, offsetValueNode As SyntaxNode, countValueNode As SyntaxNode) As Boolean
+
+            ' First argument should be an identifier name node
+            Dim firstArgumentIdentifierName As IdentifierNameSyntax = TryCast(bufferValueNode, IdentifierNameSyntax)
```

Comment:
[optional] You could get rid of the various `As XXX` as you are doing a direct assignment.

```suggestion
            Dim firstArgumentIdentifierName = TryCast(bufferValueNode, IdentifierNameSyntax)
```

Reply:
Nice. I'll do that change in this whole file.
This actually sounds like a candidate for a Roslyn analyzer 😄. Visual Studio is currently not giving me that suggestion.
If no one objects, I'll create the issue with this suggestion later today.